agent/proto/agent.proto

```diff
@@ -0,0 +1,209 @@
+syntax = "proto3";
+option go_package = "github.com/coder/coder/v2/agent/proto";
+
+package coder.agent.v2;
+
+import "tailnet/proto/tailnet.proto";
+
+message WorkspaceApp {
+	bytes uuid = 1;
+	string url = 2;
+	bool external = 3;
+	string slug = 4;
+	string display_name = 5;
+	string command = 6;
+	string icon = 7;
+	bool subdomain = 8;
+	string subdomain_name = 9;
+
+	enum SharingLevel {
+		SHARING_LEVEL_UNSPECIFIED = 0;
+		OWNER = 1;
+		AUTHENTICATED = 2;
+		PUBLIC = 3;
+	}
+	SharingLevel sharing_level = 10;
+
+	message Healthcheck {
```

Comment:
I don't know if we can/want to make this change, but this sticks out among all the other pascal casing. 😂

```suggestion
	message HealthCheck {
```

Reply:
No time like a refactor to fix mistakes of the past